src/obs-ndi-source.cpp

```diff
@@ -315,6 +337,17 @@ void ndi_source_getdefaults(obs_data_t *settings)
 	obs_data_set_default_bool(settings, PROP_AUDIO, true);
 }
 
+void ndi_source_ptz(struct ndi_source *s, obs_data_t *settings)
+{
+	if (!obs_data_get_bool(settings, PROP_PTZ))
+		return;
+	float pan = (float)obs_data_get_double(settings, PROP_PAN);
```

Comment:
Should there be any range checks here?
Will NDI ptz accept bad values and not bork?

Reply:
the min and max are set in the properties, but we can add an extra check here